gui/gui/src/TGView.cxx

```diff
@@ -464,7 +464,10 @@ void TGView::ScrollToPosition(TGLongPosition pos)
 void TGView::ScrollCanvas(Int_t new_top, Int_t direction)
 {
    Point_t points[4];
-   Int_t xsrc, ysrc, xdest, ydest, cpyheight, cpywidth;
+   Int_t xsrc, ysrc, xdest, ydest;
+   #ifndef R__HAS_COCOA
+   Int_t cpyheight, cpywidth;
```

Comment:
Given that some code paths below assign this to `0`, maybe we could just remove those lines below and initialize it here instead.
```suggestion
   Int_t cpyheight = 0, cpywidth = 0;
```